sys_plugin/vim/vimize.bas

```diff
@@ -29,16 +29,25 @@ Private Sub SetKeyMapping()'{{{
   Call nmap("j", "move_down")
   Call nmap("k", "move_up")
   Call nmap("l", "move_right")
+  Call nmap("H", "move_left_5")
+  Call nmap("J", "move_down_5")
+  Call nmap("K", "move_up_5")
+  Call nmap("L", "move_right_5")
   Call nmap("gg", "gg")
-  Call nmap("G", "G")

```

Comment:
why "G" command mapping removed?

Reply:
Because I'm a fool mostly. That one was accidental!


See latest commit: 
- Added G back
- xlam is updated
- Simplified yank to just use v_y